trieste/models/gpflux/architectures.py

```diff
@@ -0,0 +1,175 @@
+
+#
+# Copyright (c) 2021 The GPflux Contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+This module provides :func:`build_constant_input_dim_deep_gp` to build a Deep GP of
+arbitrary depth where each hidden layer has the same input dimensionality as the data.
+"""
+
+from dataclasses import dataclass
+from typing import cast
+
+import numpy as np
+import tensorflow as tf
+from scipy.cluster.vq import kmeans2
+
+import gpflow
+from gpflow.kernels import SquaredExponential
+from gpflow.likelihoods import Gaussian
+
+from gpflux.helpers import (
+    construct_basic_inducing_variables,
+    construct_basic_kernel,
+    construct_mean_function,
+)
+from gpflux.layers.gp_layer import GPLayer
+from gpflux.layers.likelihood_layer import LikelihoodLayer
+from gpflux.models import DeepGP
+
+
+@dataclass
+class Config:
```

Comment:
Config seems like a copy-paste, do we need to redefine it here?

Reply:
no..it should just be an import